### ISSUE TITLE
"Genshin window could not be focused. Please make sure the game is visible."

### ISSUE THREAD

**Issue by ironymo** [Andrewthe13th/Inventory_Kamera#255]
I get this error. I have checked the setup 27 times and everything is correct to the t.  I have deleted and re-installed the camera 4 times but I get the same error no matter what I do.

**Comment by ironymo** [comment]
It opens other random windows to apps like Realtek Jack Windows 24, ETDCrtl, Waves MaxxAudio Service Application,  Skype etc.

**Comment by wannabepudge** [comment]
I am having the same issue. Every time I click to start scan, my game minimizes. I can't figure out how to keep it up and use the utility.

**Comment by Divyaansha** [comment]
You need to keep the game open and in the menu mode. Also, the game need to be in the Windowed Mode under settings > graphics. If it is not in Windowed mode, it will throw that error (from what I experienced). There are steps mentioned in the instructions page stating it is required..

**Comment by Divyaansha** [comment]
I just finished the scan, and did not face this issue. I had this issue before, but I noticed that I missed the step that mentioned that the app needs to be in a Windowed mode. The app seems to mimic mouse and keyboard, hence it is not advised to move or touch either of them while it is working. I have the game installed via Epic store on the PC. Maybe mentioning more details about your installation might help fixing your issue.
